docs/pages/toggler.md

```diff
@@ -8,7 +8,7 @@ video: 'wHpZCrpKlBc'
 
 ## Toggle a CSS class
 
-To setup a class toggle, start by adding the attribute `data-toggler` to an element. The value of `data-toggler` is the class you want to toggle. Also give the element a unique ID so it can be targeted.
+To setup a class toggle, start by adding the attribute `data-toggler` to an element. The value of `data-toggler` is the class you want to toggle (supports `.classname` and `classname`). Also give the element a unique ID so it can be targeted.
```

Comment:
What about 

> (like `.classname` and `classname`)

Sounds more consistent with a "familiar" description of the component options, rather than a strict settings reference.

Reply:
Done.